### PR TITLE
Added some ExampleMod comments

### PR DESCRIPTION
**Pull request by RyanMakesMods**
### What are the changes to ExampleMod?
I added a lot of comments that make ExampleMod easier to read for people who look at the repository.
The comments are now simpler and are friendly for new modders.
### Do these changes need additional documentation?
Potentially, if any contributor sees fit, then yes.

### CODE REVIEW

File: ExampleMod/Content/Items/Accessories/ExampleShield.cs [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 namespace ExampleMod.Content.Items.Accessories
 {
-	[AutoloadEquip(EquipType.Shield)]
+	[AutoloadEquip(EquipType.Shield)] // load sprite as a shield when eqipped on the player
```
Comment (Mirsario) [inline ExampleMod/Content/Items/Accessories/ExampleShield.cs:10]: Please use capitalization and periods consistently in your comments. There are also inconsistencies in spacing after double forward slashes in same files.

File: ExampleMod/Content/Items/ExampleQuestFish.cs [hunk 2]
```diff
@@ -16,16 +16,16 @@ public class ExampleQuestFish : ModItem
...
+		public override bool IsQuestFish() => true; // makes the item a quest fish
 
-		public override bool IsAnglerQuestAvailable() => Main.hardMode;
+		public override bool IsAnglerQuestAvailable() => Main.hardMode; // makes the quest only appear in hard mode
 
 		public override void AnglerQuestChat(ref string description, ref string catchLocation) {
-			description = "I've heard stories of a fish that swims upside-down. Supposedly you have the stand upside-down yourself to even find one. One of those would go great on my ceiling. Go fetch!";
-			catchLocation = "Caught anywhere while standing upside-down.";
+			/* how the angler describes the fish to the player*/ description = "I've heard stories of a fish that swims upside-down. Supposedly you have the stand upside-down yourself to even find one. One of those would go great on my ceiling. Go fetch!";
+			/* what it says on the bottom of the angler's text box of how to catch the fish */ catchLocation = "Caught anywhere while standing upside-down.";
```
Comment (Mirsario) [inline ExampleMod/Content/Items/ExampleQuestFish.cs:28]: Use single-line comments on top long lines, not in front of them.

File: ExampleMod/Content/Items/ExampleSoul.cs [hunk 3]
```diff
@@ -12,30 +12,30 @@ public class ExampleSoul : ModItem
...
+			ItemID.Sets.ItemIconPulse[item.type] = true; // the item pulses while in the player's inventory
+			ItemID.Sets.ItemNoGravity[item.type] = true; // makes the item have no gravity
 		}
 
 		public override void SetDefaults() {
 			item.width = 18;
 			item.height = 18;
 			item.maxStack = 999;
-			item.value = 1000;
+			item.value = 1000; //
```
Comment (Mirsario) [inline ExampleMod/Content/Items/ExampleSoul.cs:25]: //

File: ExampleMod/Content/Items/Weapons/ExampleMagicWeapon.cs [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+			item.mana = 11; //this is how much mana the item uses
+		}
+		public override void AddRecipes()
+		{
+        CreateRecipe(51)
+            .AddIngredient(ItemID.FallenStar, 5)
+            .AddIngredient<ExampleItem>()
+            .AddTile<ExampleWorkbench>()
+            .Register();
+
```
Comment (Mirsario) [inline ExampleMod/Content/Items/Weapons/ExampleMagicWeapon.cs:43]: This file has a lot of weird spacing, and this empty line. Use tabs.

**Comment by Jofairden** [comment]
Agreed with Mirsario's comments
If we're going for consistency, I'd want `<Code here> // Capitalized and spaced`
Also use mline notation if you're gonna use the asterisk character
```
/***
   * Multiline
   * comment
 ***/

**Comment by RyanMakesMods** [comment]
Ok, I will work on that in the next few commits.
Thanks for telling me.

File: ExampleMod/Content/Items/Weapons/ExampleMagicWeapon.cs [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+			item.shootSpeed = 7; // How fast the item shoots the projectile.
+			item.crit = 32; // The percent chance at hitting an enemy with a crit, plus the default amount.
+			item.mana = 11; // This is how much mana the item uses.
+		}
+		public override void AddRecipes() {
+        CreateRecipe(51)
+            .AddIngredient(ItemID.FallenStar, 5)
+            .AddIngredient<ExampleItem>()
+            .AddTile<ExampleWorkbench>()
+            .Register();
```
Comment (Mirsario) [inline ExampleMod/Content/Items/Weapons/ExampleMagicWeapon.cs:39]: You did not fix the space/tab issue.

**Pull request merged**